PyInstaller/compat.py

```diff
@@ -171,9 +172,9 @@ def encode(text):
 # compat.base_prefix with the path to the
 # base Python installation.
 
-base_prefix = getattr( sys, 'real_prefix',
-                       getattr( sys, 'base_prefix', sys.prefix )
-                        )
+base_prefix = getattr(sys, 'real_prefix',
+                      getattr(sys, 'base_prefix', sys.prefix)
+                      )
```

Comment:
Please abstain from reformatting. If any, the have to go into a separate commit, see development guideline.